backend/src/models/open-source-project.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
It looks like this file is a copy of the other `open_source_project.ts` file (likely a bug due to changing of the name), so this one can probably be deleted.